charts/ps-db/templates/cluster.yaml

```diff
@@ -474,7 +512,6 @@ spec:
     {{- end }}
 
   orchestrator:
-    size: {{ .Values.orchestrator.size }}
```

Comment:
Size is required in CR:

`Error: 1 error occurred:
        * PerconaServerMySQL.ps.percona.com "db-ps-db" is invalid: [spec.orchestrator.size: Required value, <nil>: Invalid value: "null": some validation rules were not checked because the object was invalid; correct the existing errors to complete validation]`

Reply:
Yes, when the feature is enabled, it's necessary to define size when feature is enabled and disabled cause it's not an optional field. It's possible to double check it deploying with orchestrator disabled.

Reply:
https://github.com/percona/percona-helm-charts/pull/669/commits/e26aff496adc36f8e454dcb9724b3a3fa2286a3b